thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/controller.js

```diff
@@ -416,7 +416,7 @@ export default Ember.Controller.extend({
       const granularity = this.get('graphConfig.granularity').toLowerCase();
       const settingsByGranularity = {
         common: {
-          functionName: this.get('alertFunctionName'),
+          functionName: this.get('alertFunctionName').trim(),
```

Comment:
perhaps use the same logic as below `this.get('alertFunctionName') || this.get('alertFunctionName').trim()`